constraint/pkg/externaldata/response.go

```diff
@@ -0,0 +1,51 @@
+package externaldata
+
+// RegoResponse is the response inside rego.
+type RegoResponse struct {
+	Responses   [][]interface{} `json:"responses"`
+	Errors      [][]interface{} `json:"errors"`
+	StatusCode  int             `json:"status_code"`
+	SystemError string          `json:"system_error"`
+}
+
+// ProviderResponse is the API response from a provider.
+type ProviderResponse struct {
+	APIVersion string   `json:"apiVersion,omitempty"`
+	Kind       string   `json:"kind,omitempty"`
+	Response   Response `json:"response,omitempty"`
+}
+
+// Response is the struct that holds the response from a provider.
+type Response struct {
+	Idempotent  bool   `json:"idempotent,omitempty"`
```

Comment:
Should we comment on these fields as well?

Reply:
added